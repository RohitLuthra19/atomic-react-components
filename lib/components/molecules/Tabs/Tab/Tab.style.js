import { css } from 'styled-components';

export default css`
  padding: 10px 20px;
  border: 1px solid ${props => props.theme.borderColorSecondary};
  ${props => (props.inheritedStyles ? props.inheritedStyles : '')};
`;
