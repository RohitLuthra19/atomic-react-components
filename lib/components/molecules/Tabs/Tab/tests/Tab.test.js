import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer';

import Tab from '../index';

describe('<Tab />', () => {
  let TabComponent = '';
  beforeEach(() => {
    TabComponent = shallow(<Tab>Test</Tab>);
  });

  test('should render correctly', () => {
    expect(TabComponent).toMatchSnapshot();
  });
});
