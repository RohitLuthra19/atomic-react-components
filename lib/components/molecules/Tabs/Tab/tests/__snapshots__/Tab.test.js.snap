// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tab /> should render correctly 1`] = `
<Tab
  className="Tab-we9qei-0 ePIzqc"
  inheritedStyles=""
>
  Test
</Tab>
`;
